packages/components/inputs/localized-multiline-text-input/src/localized-multiline-text-input.visualroute.jsx

```diff
@@ -155,5 +155,25 @@ export const component = () => (
         hasWarning={true}
       />
     </Spec>
+    <Spec label="when there is a additional info set for a locale">
```

Comment:
I think we need new specs to validate all the possible combinations:
![image](https://github.com/commercetools/ui-kit/assets/97907068/c5796b4b-1ae0-4ac0-bcc2-65c09b4dd5ea)

Reply:
Actually I found what I think is an unexpected behaviour in the `LocalizedRichTextInput`.

If we have additional messages for all the languages but they are collapsed (only english one is rendered), we can see the additional message is right below the text editor:
![image](https://github.com/commercetools/ui-kit/assets/97907068/8bdc55cd-ed3f-44df-8552-6b951b7d5e99)

But if we show all the languages, now the first two additional messages are rendered like in the second row, but the third editor shows the additional message right below the editor:
![image](https://github.com/commercetools/ui-kit/assets/97907068/34ce936a-7683-4272-ac27-10a1073024ae)

Reply:
Good catch! I pushed a fix for that